jdaviz/core/template_mixin.py

```diff
@@ -2871,8 +2872,8 @@ def _get_continuum(self, dataset, spatial_subset, spectral_subset, update_marks=
                           'center': spectral_axis.value,
                           'right': []}
             else:
-                mark_x = {'left': spectral_axis_nanmasked[spectral_axis.value < sr_lower.value],
-                          'right': spectral_axis_nanmasked[spectral_axis.value > sr_upper.value]}
+                mark_x = {'left': spectral_axis_nanmasked[spectral_axis < sr_lower],
```

Comment:
I am unable to tell from a quick glance if `mark_x` is going into any front-end message. Looks like you are now passing Quantity instead of float arrays? Is the front-end JSON thingy able to handle this without warning?

Reply:
From my testing it is, did you experience a bug relating to this?

Reply:
In the past, I had seen warnings polluting the notebook cell outputs when JSON is trying to serialize something fancy like Quantity. But if you don't see it, then maybe it is not a problem here. Just double checking. Thanks!

Reply:
The quantity is just being used to determine the indices though, right?  `spectral_axis_nanmasked` is unchanged here.